content/tutorials/queue-stomp-neutrino-profanity.textile

```diff
@@ -233,7 +233,7 @@ We're now ready to start consuming messages from the queue. There are "plenty of
 
 ```[json]
   "dependencies": {
-    "ably": ">=0.9.0-beta",
+    "ably": "~1.0",
```

Comment:
Well we previously locked into minor versions, I don't really see the harm, but sure I can change to `~1`